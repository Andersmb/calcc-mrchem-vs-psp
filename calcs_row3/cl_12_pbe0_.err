The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) StdEnv
Lmod has detected the following error: Your site prevents the automatic
swapping of modules with same name. You must explicitly unload the loaded
version of "OpenMPI/4.1.1-GCC-11.2.0" before you can load the new one. Use swap
to do this:

   $ module swap OpenMPI/4.1.1-GCC-11.2.0 OpenMPI/4.0.3-GCC-9.3.0

Alternatively, you can set the environment variable
LMOD_DISABLE_SAME_NAME_AUTOSWAP to "no" to re-enable same name autoswapping.



Lmod has detected the following error: Your site prevents the automatic
swapping of modules with same name. You must explicitly unload the loaded
version of "GCCcore/11.2.0" before you can load the new one. Use swap to do
this:

   $ module swap GCCcore/11.2.0 GCCcore/9.3.0

Alternatively, you can set the environment variable
LMOD_DISABLE_SAME_NAME_AUTOSWAP to "no" to re-enable same name autoswapping.

While processing the following module(s):
    Module fullname             Module Filename
    ---------------             ---------------
    Python/3.8.2-GCCcore-9.3.0  /cluster/modulefiles/all/Python/3.8.2-GCCcore-9.3.0.lua

cp: cannot stat ‘checkpoint/*’: No such file or directory
