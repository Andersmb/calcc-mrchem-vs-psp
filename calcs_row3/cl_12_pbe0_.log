Starting job 5688109 on c1-[6,18,24],c5-[11,25] at Thu May 19 20:07:53 CEST 2022

Copying cl_12_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying cl_12_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688109      cl_12_pbe+        180                                             01:14:46      0:0 
5688109.bat+      batch         36        1   00:00:03          0   00:00:03   01:14:46      0:0 
5688109.ext+     extern        180        5   00:00:00          1   00:00:00   01:14:46      0:0 
5688109.0      mrchem.x        180       30   01:13:18         29   03:05:59   01:13:25      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688109                                                                          
5688109.bat+     19284K          0     19284K        0              0          0 
5688109.ext+          0          1          0        0              1          0 
5688109.0     17815144K         29 6976932113        0             22          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688109                                                                                               
5688109.bat+       19.39M               0         19.39M        0.07M                0          0.07M 
5688109.ext+        0.00M               1          0.00M            0                1              0 
5688109.0           6.77M              15          6.75M      149.33M               11         84.76M 

Job 5688109 completed at Thu May 19 21:22:39 CEST 2022
