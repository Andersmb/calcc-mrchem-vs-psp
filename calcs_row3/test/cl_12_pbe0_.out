                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :       -0.616555        0.000584       -0.287484
    1       O            :        0.109478       -0.012077        2.999127
    2       O            :        0.567926       -2.332283       -1.288661
    3       O            :        0.565847        2.342899       -1.267496
    4       H            :        1.951772        0.002662        3.080385
---------------------------------------------------------------------------
 Center of mass          :       -0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              21
 Beta electrons          :                                              21
 Total electrons         :                                              42
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999787439e-01
    1    1    a          :                              1.000000041745e+00
    2    1    a          :                              1.000000156124e+00
    3    1    a          :                              1.000000002496e+00
    4    1    a          :                              1.000000023485e+00
    5    1    a          :                              1.000000002782e+00
    6    1    a          :                              9.999999947022e-01
    7    1    a          :                              9.999999994737e-01
    8    1    a          :                              9.999998114395e-01
    9    1    a          :                              9.999997316589e-01
   10    1    a          :                              9.999999710693e-01
   11    1    a          :                              9.999993214621e-01
   12    1    a          :                              9.999991731193e-01
   13    1    a          :                              1.000000770358e+00
   14    1    a          :                              9.999997208868e-01
   15    1    a          :                              9.999996207838e-01
   16    1    a          :                              1.000001176773e+00
   17    1    a          :                              9.999999704328e-01
   18    1    a          :                              1.000000325517e+00
   19    1    a          :                              1.000001279645e+00
   20    1    a          :                              1.000000188202e+00
   21    1    b          :                              9.999998773746e-01
   22    1    b          :                              1.000000013368e+00
   23    1    b          :                              9.999997997621e-01
   24    1    b          :                              1.000000004376e+00
   25    1    b          :                              1.000000001428e+00
   26    1    b          :                              1.000000002812e+00
   27    1    b          :                              9.999999947022e-01
   28    1    b          :                              9.999999995854e-01
   29    1    b          :                              9.999998100281e-01
   30    1    b          :                              1.000000019843e+00
   31    1    b          :                              9.999999701256e-01
   32    1    b          :                              9.999994589080e-01
   33    1    b          :                              9.999989128356e-01
   34    1    b          :                              1.000000908647e+00
   35    1    b          :                              9.999997086671e-01
   36    1    b          :                              9.999997052721e-01
   37    1    b          :                              1.000001176785e+00
   38    1    b          :                              9.999999716300e-01
   39    1    b          :                              1.000000324273e+00
   40    1    b          :                              1.000001294259e+00
   41    1    b          :                              1.000000511866e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                680.216028892460
 E-N energy              :            (au)              -1992.328092613122
 Coulomb energy          :            (au)                495.639577953037
 Exchange energy         :            (au)                -13.176183525064
 X-C energy              :            (au)                -40.986233559095
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                188.810894895964
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -870.634902851783
 Nuclear energy          :            (au)                188.810894895964
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.818240079558e+02
                         :      (kcal/mol)             -4.278510211838e+05
                         :        (kJ/mol)             -1.790128673795e+06
                         :            (eV)             -1.855337561682e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.882809839624
    1    1    a          :            (au)                -18.914242018041
    2    1    a          :            (au)                -18.891876187642
    3    1    a          :            (au)                -18.729428852333
    4    1    a          :            (au)                -10.222405965791
    5    1    a          :            (au)                 -7.971629605238
    6    1    a          :            (au)                 -7.970376094089
    7    1    a          :            (au)                 -7.963247747638
    8    1    a          :            (au)                 -1.408744803749
    9    1    a          :            (au)                 -1.093659337723
   10    1    a          :            (au)                 -1.034188390518
   11    1    a          :            (au)                 -0.853774784686
   12    1    a          :            (au)                 -0.646007687243
   13    1    a          :            (au)                 -0.501481886891
   14    1    a          :            (au)                 -0.428582523936
   15    1    a          :            (au)                 -0.389169064266
   16    1    a          :            (au)                 -0.280455402088
   17    1    a          :            (au)                 -0.255785018670
   18    1    a          :            (au)                 -0.234948752419
   19    1    a          :            (au)                 -0.186910293058
   20    1    a          :            (au)                 -0.179886136062
   21    1    b          :            (au)               -101.882827383824
   22    1    b          :            (au)                -18.914235487854
   23    1    b          :            (au)                -18.891867453255
   24    1    b          :            (au)                -18.729449076807
   25    1    b          :            (au)                -10.222410267914
   26    1    b          :            (au)                 -7.971622545772
   27    1    b          :            (au)                 -7.970374328362
   28    1    b          :            (au)                 -7.963250326061
   29    1    b          :            (au)                 -1.408747322966
   30    1    b          :            (au)                 -1.093659357818
   31    1    b          :            (au)                 -1.034185000888
   32    1    b          :            (au)                 -0.853769842201
   33    1    b          :            (au)                 -0.646000758540
   34    1    b          :            (au)                 -0.501486293472
   35    1    b          :            (au)                 -0.428582284730
   36    1    b          :            (au)                 -0.389172721755
   37    1    b          :            (au)                 -0.280455230172
   38    1    b          :            (au)                 -0.255784627338
   39    1    b          :            (au)                 -0.234949033011
   40    1    b          :            (au)                 -0.186911785682
   41    1    b          :            (au)                 -0.179888708864
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -400.079240228991
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          6.480741e+00        -681.824007955820         -6.818240e+02
    1          5.793866e-01        -685.722925234026         -3.898917e+00
    2          2.520148e-01        -685.921809288791         -1.988841e-01
    3          1.362889e-01        -685.967538015241         -4.572873e-02
    4          7.047798e-02        -685.984249461063         -1.671145e-02
    5          9.293660e-02        -686.003510526142         -1.926107e-02
    6          2.581783e-02        -686.005986572703         -2.476047e-03
    7          2.814359e-02        -686.007870707736         -1.884135e-03
    8          1.357010e-02        -686.008218626307         -3.479186e-04
    9          1.848953e-02        -686.008510966331         -2.923400e-04
   10          8.358807e-03        -686.008642838197         -1.318719e-04
   11          9.074292e-03        -686.008730141720         -8.730352e-05
   12          4.030329e-03        -686.008757905165         -2.776345e-05
   13          3.678467e-03        -686.008776997683         -1.909252e-05
   14          3.366144e-03        -686.008791153431         -1.415575e-05
   15          2.913614e-03        -686.008798846158         -7.692727e-06
   16          1.618419e-03        -686.008801573033         -2.726875e-06
   17          1.289520e-03        -686.008803642339         -2.069307e-06
   18          8.326313e-04        -686.008804529247         -8.869075e-07
   19          6.005383e-04        -686.008805002765         -4.735184e-07
   20          5.609492e-04        -686.008805280865         -2.780999e-07
   21          3.690024e-04        -686.008805410300         -1.294346e-07
   22          1.982791e-04        -686.008805477308         -6.700793e-08
   23          1.663854e-04        -686.008805529907         -5.259915e-08
---------------------------------------------------------------------------
                      SCF converged in 23 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :       -0.616555        0.000584       -0.287484
    1       O            :        0.109478       -0.012077        2.999127
    2       O            :        0.567926       -2.332283       -1.288661
    3       O            :        0.565847        2.342899       -1.267496
    4       H            :        1.951772        0.002662        3.080385
---------------------------------------------------------------------------
 Center of mass          :       -0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                684.309131273176
 E-N energy              :            (au)              -2003.863414594812
 Coulomb energy          :            (au)                498.625398338988
 Exchange energy         :            (au)                -13.085883027409
 X-C energy              :            (au)                -40.804932415813
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                188.810894895964
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -874.819700425870
 Nuclear energy          :            (au)                188.810894895964
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.860088055299e+02
                         :      (kcal/mol)             -4.304770212874e+05
                         :        (kJ/mol)             -1.801115858235e+06
                         :            (eV)             -1.866724975496e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.996340002999
    1    1    a          :            (au)                -19.281631836490
    2    1    a          :            (au)                -19.277542684231
    3    1    a          :            (au)                -19.277533199911
    4    1    a          :            (au)                 -9.788163580037
    5    1    a          :            (au)                 -7.522543368682
    6    1    a          :            (au)                 -7.520909701547
    7    1    a          :            (au)                 -7.515041760388
    8    1    a          :            (au)                 -1.258230273785
    9    1    a          :            (au)                 -1.096899813880
   10    1    a          :            (au)                 -1.074838161769
   11    1    a          :            (au)                 -0.790664864862
   12    1    a          :            (au)                 -0.613637107316
   13    1    a          :            (au)                 -0.582266823602
   14    1    a          :            (au)                 -0.563112698700
   15    1    a          :            (au)                 -0.495640185775
   16    1    a          :            (au)                 -0.419965607449
   17    1    a          :            (au)                 -0.402750434682
   18    1    a          :            (au)                 -0.382778928339
   19    1    a          :            (au)                 -0.355381935384
   20    1    a          :            (au)                 -0.346772892351
   21    1    b          :            (au)               -101.996340002082
   22    1    b          :            (au)                -19.281631806181
   23    1    b          :            (au)                -19.277542698587
   24    1    b          :            (au)                -19.277533213213
   25    1    b          :            (au)                 -9.788163578831
   26    1    b          :            (au)                 -7.522543370187
   27    1    b          :            (au)                 -7.520909700401
   28    1    b          :            (au)                 -7.515041760274
   29    1    b          :            (au)                 -1.258230280946
   30    1    b          :            (au)                 -1.096899843920
   31    1    b          :            (au)                 -1.074838103692
   32    1    b          :            (au)                 -0.790664871030
   33    1    b          :            (au)                 -0.613637099282
   34    1    b          :            (au)                 -0.582266835687
   35    1    b          :            (au)                 -0.563112711639
   36    1    b          :            (au)                 -0.495640168984
   37    1    b          :            (au)                 -0.419965638476
   38    1    b          :            (au)                 -0.402750408773
   39    1    b          :            (au)                 -0.382778969516
   40    1    b          :            (au)                 -0.355381946390
   41    1    b          :            (au)                 -0.346772898268
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -401.125291768537
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -1.476353        0.001863       -1.378765
 Magnitude               :            (au)                        2.020053
                         :         (Debye)                        5.134463
---------------------------------------------------------------------------
 Nuclear vector          :        1.416345        0.000896        1.736911
 Magnitude               :            (au)                        2.241181
                         :         (Debye)                        5.696513
---------------------------------------------------------------------------
 Total vector            :       -0.060009        0.002759        0.358146
 Magnitude               :            (au)                        0.363149
                         :         (Debye)                        0.923033
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  1h 13m 20s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
